### ISSUE TITLE
Testing data sent to YNAB API 

### ISSUE THREAD

**Issue by nocalla** [bank2ynab/bank2ynab#263]
**Other feature request**

**Is your feature request related to a problem? Please describe.**
Travis can't upload things to his own budget (I'm sure he's very fiscally responsible anyway, he seems like the kind of guy who relentlessly counts every cent). For our automated testing we need to ensure that the API is receiving data in the appropriate format.

**Describe the solution you'd like**
I'd like to include a simulation of the API's limitations in our testing suite that we can bounce our data off.

**Describe alternatives you've considered**
We could just not test this, but that seems irresponsible as it may result in data corruption in users' budgets. 

**Additional context**
_We need to create a list of the limitations of the API here - in progress!_

- Payee name limit: 50 characters
- Memo limit: 100 characters
- Import_id limit: 36 characters

(Source: https://support.youneedabudget.com/t/80bp6b)

**Comment by torbengb** [comment]
I think we need to ask YNAB Support about those limits. It's unlikely that
we can black-box identify them all on our own.

**Comment by nocalla** [comment]
I've sent a question through the app, will update if they respond.

**Comment by nocalla** [comment]
Got a response from George on the API team. 

"As of next Tuesday there shouldn't be any more undocumented limitations on transaction fields.

I just added documentation for the validation constraints for creating transactions.

payee_name has a max length of 50 characters
memo is 100 max
import_id is 36 max
I also fixed a minor copy issue with the import_id validation message. It used to read:

import is too long (maximum is 36 characters)

Now it reads:

import_id is too long (maximum is 36 characters)

I'll post here when it hits production. Cheers"
